src/sentry/static/sentry/app/views/alerts/wizard/index.tsx

```diff
@@ -163,10 +163,9 @@ const StyledPageHeader = styled(PageHeader)`
   margin-bottom: ${space(4)};
 `;
 
-const Heading = styled('h1')`
-  font-size: ${p => p.theme.fontSizeExtraLarge};
-  font-weight: normal;
-  margin-bottom: ${space(1)};
+const StyledPlaceholder = styled(Placeholder)`
```

Comment:
I tend to like giving these more semantic names. Like could this be 'WizardBodyPlaceholder` ?